client/src/components/logInForm.js

```diff
@@ -0,0 +1,88 @@
+import React, { useContext, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Form, Input, Button, Alert } from 'antd';
+import axios from 'axios';
+import IsAuthContext from './context/isAuthContext';
```

Comment:
If You make the context like [this](https://github.com/GSG-G9/dentoro/pull/62/files#r609413886)

Then here you only need to import useAuth 
```js
import { useAuth } from './context/isAuthContext';
```

and then use it like this :)
```js
  const [, setIsAuth] = useAuth();
```